src/dataMap/metaManager/metaSchema.js

```diff
@@ -287,8 +287,14 @@ export default () => {
      * (if you want to define column width separately for each column) or a function (if you want to set column width
      * dynamically on each render).
      *
+     * The default width for columns in the rendering process equals 50px.
+     *
+     * An `undefined` value is for detection in {@link Hooks#modifyColWidth} hook if plugin or setting changed the default size.
+     *
+     * Note: This option will forcely disable {@link AutoColumnSize} plugin.
+     *
      * @memberof Options#
-     * @type {number|number[]|string|string[]|Function}
+     * @type {number|number[]|string|string[]|undefined[]|Function}
```

Comment:
When I try to run `jsdoc` this file can't be parsed and throw an error in this line. After a change to `Array<undefined>` it's ok.

```suggestion
     * @type {number|number[]|string|string[]|Array<undefined>|Function}
```